types/stripe-js/index.d.ts

```diff
@@ -341,6 +341,24 @@ declare module '@stripe/stripe-js' {
       data?: ConfirmBacsDebitSetupData
     ): Promise<{setupIntent?: SetupIntent; error?: StripeError}>;
 
+    /**
+     * Use `stripe.confirmBancontactSetup` in the [Set up future payments](https://stripe.com/docs/payments/bancontact/set-up-payment) flow to use Bancontact bank details to set up a SEPA Direct Debit payment method for future payments.
+     * When called, it will confirm a `SetupIntent` with `data` you provide, and it will automatically redirect the customer to authorize the transaction.
+     * Once authorization is complete, the customer will be redirected back to your specified `return_url`.
+     * Note that there are some additional requirements to this flow that are not covered in this reference.
+     * Refer to our [integration guide](https://stripe.com/docs/payments/bancontact/set-up-payment) for more details.
+     *
+     * When you confirm a `SetupIntent`, it needs to have an attached [PaymentMethod](https://stripe.com/docs/api/payment_methods).
+     * In addition to confirming the `SetupIntent`, this method can automatically create and attach a new `PaymentMethod` for you.
+     * It can also be called with an existing `PaymentMethod`, or if you have already attached a `PaymentMethod` you can call this method without needing to provide any additional data.
+     *
+     * @docs https://stripe.com/docs/js/payment_intents/confirm_bancontact_setup
+     */
+    confirmBancontactSetup(
+      clientSecret: string,
+      data?: ConfirmBancontactSetupData
+    ): Promise<{paymentIntent?: SetupIntent; error?: StripeError}>;
```

Comment:
```suggestion
    ): Promise<{setupIntent?: SetupIntent; error?: StripeError}>;
```
Might be worth adding to the type tests.

Reply:
Oh boy, what a bad copy paste!